lib/v2/hackyournews/index.js

```diff
@@ -0,0 +1,50 @@
+// Require necessary modules
+const got = require('@/utils/got'); // a customised got
+const cheerio = require('cheerio'); // an HTML parser with a jQuery-like API
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const baseUrl = 'https://hackyournews.com';
+    const { data: response } = await got(String(baseUrl));
```

Comment:
```suggestion
    const { data: response } = await got(baseUrl);
```
`baseUrl` is a string already, no need to cast it again.